### PR TITLE
Minor changes to subtle interactions of the solver that can't possibly affect anything

### PR DESCRIPTION
**Pull request by willkelly**

### CODE REVIEW

**Comment by breu** [comment]
please review your use of aggressive line wrapping

File: opencenter.conf [hunk 1]
```diff
@@ -1,4 +1,5 @@
 # default values are commented in each section
+# one last change
```
Comment (breu) [inline opencenter.conf:2]: Is this the best place for this?

**Comment by rpedde** [comment]
passive-aggressive, that is.

File: opencenter.conf (hunk 1 above)
Comment (rpedde) [inline opencenter.conf:2]: I feel like there should be another line feed here.

Comment (galstrom21) [inline opencenter.conf:2]: There should be at least one blank line above this commit (pep8 violation)

**Comment by rpedde** [comment]
-1 please don't commit this

**Comment by breu** [comment]
-1 I Would prefer that you didn't submit this

**Comment by willkelly** [comment]
I CURSE YOU

**Comment by Apsu** [comment]
Wait, I didn't push my changes!

**Comment by breu** [comment]
merged

File: opencenter.conf (hunk 1 above)
Comment (breu) [inline opencenter.conf:2]: and probably one after...  because... you know...

**Pull request closed without merging**